.mise/tasks/install

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
-# mise description="Performs additional tasks that are necessary to work in this repository"
+# mise description="Install all necessary dependencies"
 
 set -euo pipefail
 
-pnpm install -C docs/
\ No newline at end of file
+pnpm install -C $MISE_PROJECT_ROOT/docs/
+swift package --package-path $MISE_PROJECT_ROOT resolve
+swift package --package-path $MISE_PROJECT_ROOT/app resolve
```

Comment:
Out of curiosity, what motivated the creation of another `Project` instead of having a single one?

Reply:
I see the Tuist App and the Tuist CLI as separate projects. If we do decide to move it away into a different repo, having that distinction early on makes things easier.